src/object.d

```diff
@@ -1295,6 +1637,18 @@ class TypeInfo_Interface : TypeInfo
     }
 }
 
+@safe unittest
```

Comment:
If this is `@safe`, what prevents other unittests that call `toString`/`typeid` to also be `@safe`?

Reply:
Yeah, here is actually possible to call toString under @safe, for example for Enum `typeid()`. Changed that unittest. Although as I stated above, I'll later change TypeInfo to be @safe on some methods that currently are not.